lib/engine/game/g_18_cz/game.rb

```diff
@@ -2920,6 +2920,17 @@ def progress_information
             { type: :End },
           ]
         end
+
+        def route_distance(route)
+          return super if train_of_size?(route.train, :small)
+
+          n_cities = route.stops.select { |n| n.city? || n.offboard? }.size
```

Comment:
stops.count

Reply:
Since I copied the base code from 1860, I also changed the 1860 code, I hope thats fine